src/SFA.DAS.RoatpOversight.Web/Infrastructure/ApiClients/TokenService/RoatpRegisterTokenService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using SFA.DAS.RoatpOversight.Web.Settings;
+
+namespace SFA.DAS.RoatpOversight.Web.Infrastructure.ApiClients.TokenService
+{
+    public class RoatpRegisterTokenService : IRoatpRegisterTokenService
+    {
+        private readonly IWebConfiguration _configuration;
+        private readonly IHostingEnvironment _hostingEnvironment;
+        public RoatpRegisterTokenService(IWebConfiguration configuration, IHostingEnvironment hostingEnvironment)
+        {
+            _configuration = configuration;
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public string GetToken()
+        {
+            if (_hostingEnvironment.IsDevelopment())
+                return string.Empty;
```

Comment:
This doesn't work if you try to use anything other that local APIs.  IsLoopback works a lot better.

Reply:
Updated